lib/lighthouse/benefits_documents/service_exception.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
```

Comment:
It doesn't look like you are using this class at all. Safe to remove?

Reply:
You are correct. I copied it from Claims but then ended up never using it